modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-controller-nodeportservice-projects_{context}"]
+= Adding a single NodePort service to an Ingress Controller
+
+Instead of creating a `NodePort`-type `Service` for each project, you can create a single service and apply it to all of your projects. You can configure this type of environment by setting your Ingress Controller to use the `NodePortService` endpoint publishing strategy.
```

Comment:
```suggestion
Instead of creating a `NodePort`-type `Service` for each project, you can create an Ingress Controller to use the `NodePortService` endpoint publishing strategy.
```

Reply:
:heavy_check_mark: